extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -70,60 +74,34 @@ public class DefaultAccessLogReceiver implements AccessLogReceiver, Runnable, Cl
 
     private long changeOverPoint;
     private String currentDateString;
-    private boolean forceLogRotation;
 
     private final Path outputDirectory;
     private final Path defaultLogFile;
 
     private final String logBaseName;
     private final String logNameSuffix; // always starts with a '.' character
 
-    private Writer writer = null;
+    private final BufferedWriter writer;
```

Comment:
now final seems nice to have but not suitable, as need to roll files write on different location.
but test green, question impl. or tests.